src/context.rs

```diff
@@ -172,6 +174,20 @@ fn create_timetable(ntm: &navitia_model::Model, generation_period: &Period) -> T
     timetable
 }
 
+impl GtfsRT {
+    pub fn decode_feed_message(&self) -> Option<transit_realtime::FeedMessage> {
+        use bytes::IntoBuf;
+        use prost::Message;
+        match transit_realtime::FeedMessage::decode(self.data.clone().into_buf()) {
```

Comment:
minor: this is completly valid but I think it can be rewritten as (not tested though):

```rust
transit_realtime::FeedMessage::decode(self.data.clone().into_buf())
.map_err(|e|log::warn!("Unable to decode feed message, {}", e);)
.ok()

```